OptimoveSDK/optimove-sdk/src/main/java/com/optimove/android/OptimoveConfig.java

```diff
@@ -68,28 +75,57 @@ public enum InAppDisplayMode {
         PAUSED
     }
 
-    // Private constructor to discourage not using the Builder.
-    private OptimoveConfig() {
-    }
+    public enum Region {
+        EU("eu-central-2"),
```

Comment:
Do we have any tenants using this SDK against eu-1? (probably/hopefully not, worth checking?)

Maybe acceptable to say late init only for new regions.

Reply:
i hope so. no existing clients need this